bin/cibuild

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
```

Comment:
Should we bring this out into a separate PR/change? I like the direction but might be a bit more work to get it to do all the caching things we do in the ci build.

Reply:
Ah you just using this for testing for now? 👍 on keeping for that. I was modifying the docker-dev-shell but ended up committing those changes once

Reply:
SGTM, let's remove it.

Reply:
Actually finding this pretty useful for dev testing